python/src/kagent/tool_servers/_streamable_http_mcp_tool_server.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, config: StreamableHttpMcpToolServerConfig):
 
     async def discover_tools(self) -> list[Component]:
         try:
-            logger.debug(f"Discovering tools from streamable http server: {self.config}")
+            logger.debug(f"Discovering tools from streamable http server: {self._sanitize_config(self.config)}")
```

Comment:
nice! 

While you're in this code here -- there's a similar logging statement in the `_ssemcptoolserver.py` and `_stdiomcptoolserver.py` -- can you run that through the `_sanitize_config` as well?